src/UserCounterBundle/Form/UserInfoType.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace UserCounterBundle\Form;
```

Comment:
foda-se e tinhas isto aqui.... podias ter inicializado directamente e simple... tal como...
$form = $this->createForm(new UserInfoType(), $task);
ou registavas como servico e fazias... 
$form = $this->createForm('user_info_type', $blablahbla);